includes/export/abstract-wc-csv-exporter.php

```diff
@@ -330,6 +330,8 @@ public function escape_data( $data ) {
 			$data = "'" . $data;
 		}
 
+		$data = str_replace( '"', '""', $data );
```

Comment:
I believe this causes big problems when trying to import again. I got all sorts of crazy errors after trying to import an export.

<img width="787" alt="screen shot 2017-06-09 at 9 59 55 am" src="https://user-images.githubusercontent.com/7317227/26986035-c024be4a-4cfa-11e7-9d7d-906c263046da.png">

It looks like the quotes confuse the parser because it thinks they delineate a new CSV field, so we have description chunks parsed as other fields.